public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
```

Comment:
these images we dont really need. Remove references to them as well as the files